### PR TITLE
React images upload

### PR DESCRIPTION
**Pull request by nunibaranes**

### CODE REVIEW

File: src/components/common/WMImageUploader/style.module.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/* stylelint-disable selector-class-pattern */
+.wm-image-uploader-wrapper {
+  display: inherit;
+
+  &.disabled {
+    pointer-events: none;
+    opacity: 0.6;
+
+    .uploader :global(.fileContainer) {
```
Comment (dr15) [inline src/components/common/WMImageUploader/style.module.scss:9]: Have you tried using the class props that come with the package (className, buttonClassName, labelClass, errorClass)? Were they not enough to override all the styles you wanted?

File: src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        labelHtmlFor="cover-image"
+      >
+        <WMSwitch
+          className={classes['switch-field']}
+          checked={useDefaultImage} // TODO: change to SDK property
+          label="Use TeachMe’s default cover image"
+          onChange={(checked: boolean) => setUseDefaultImage(checked)}
+        />
+      </FormGroup>
+      {}
```
Comment (dr15) [inline src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx:33]: ?

File: src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      <div className={cc([classes['image-uploader'], classes['settings-section']])}>
+        <WMImageUploader
+          buttonText="+ Choose images"
+          onImageChange={onDrop}
+          imgExtension={['.jpg', '.jpeg', '.gif', '.png', '.gif']}
```
Comment (dr15) [inline src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx:38]: There's 'gif' twice

File: src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          maxFileSize={5242880}
+          tooltipTitle={`Max file size: 5mb, accepted: ${imgExtension.join(' ,')}`}
```
Comment (dr15) [inline src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx:40]: We already know the max file size is 1mb

**Review by dr15 — COMMENTED** [review]
Looks amazing! Left a few comments

File: src/components/common/WMImageUploader/style.module.scss (hunk 1 above)
Comment (nunibaranes) [reply src/components/common/WMImageUploader/style.module.scss:9]: According to the API, the additional className not helps me to override the styles. 
But I added the additional props and use our class wherever we need :)

File: src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx (hunk 2 above)
Comment (nunibaranes) [reply src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx:33]: Sorry, my mistake, thanks 😅

File: src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx (hunk 3 above)
Comment (nunibaranes) [reply src/components/Screen/CourseEditorScreen/CourseOutline/CourseSettingsTab/ImageSettings.tsx:38]: Thanks :)

**Review by dr15 — APPROVED** [review]
Approved!

**Pull request merged**